opsdroid/core.py

```diff
@@ -1,4 +1,4 @@
-"""Core components of OpsDroid."""
+="""Core components of OpsDroid."""
```

Comment:
I think you added a `=` by mistake and this is breaking the tests, could you please delete this?